data_store/aws.py

```diff
@@ -49,7 +49,7 @@ def upload_file(file: Union[IO, FileStorage], bucket: str, object_name: str, met
 
     file.seek(0)
     _S3_CLIENT.upload_fileobj(
-        file,
+        file.stream if isinstance(file, FileStorage) else file,
```

Comment:
Hmm.. not really related to the PR.
But I think `file: Union[IO, FileStorage]` could have easily been just `FileStorage`.

Reply:
Definitely it should either just be IO or FileStorage, not both, agree 👍